tests/Mailer/Handler/SendMailsForOwnershipEventHandlerTest.php

```diff
@@ -125,7 +163,7 @@ public function it_does_not_dispatch_with_an_invalid_event(): void
                 new Description('description-456')
             ));
 
-        $this->domainMessageSpecification
+        $this->domainMessageSpecificationForReplays
```

Comment:
I know that this test is not changed, but I think that the dispatch is not called because the replay mock returns `true`. So it doesn't really test the `switch` statement inside the `handle` method.